packages/core/src/input/input-file-resolver.ts

```diff
@@ -117,6 +119,14 @@ export class InputFileResolver {
   }
 
   private async expandPattern(globbingExpression: string, logAboutUselessPatterns: boolean): Promise<string[]> {
+    if (RegExp('(:\\d+){4}$').exec(globbingExpression)) {
```

Comment:
same here. Maybe consider putting this regex in a variable somewhere since it's used everywhere 🤔 easier to maintain in the future i guess